cmd/es.go

```diff
@@ -30,18 +15,18 @@ var elasticCmd = &cobra.Command{
 	Run: func(cmd *cobra.Command, args []string) {
 		ctx := context.Background()
 
-		// port := viper.GetString("port")
 		address := viper.GetString("es_address")
-		es.ElasticClient(ctx, address)
+		mapping := viper.GetString("es_mapping_file")
+		es.ElasticClient(ctx, address, mapping)
 	},
 }
 
 func init() {
 
 	rootCmd.AddCommand(elasticCmd)
 	// flag to
-	elasticCmd.Flags().StringP("es_address", "a", "localhost:9200", "url for Elasticsearch server")
-	elasticCmd.Flags().StringP("es_mapping_file", "m", "./etc/mapping.json", "location for es mapping file")
+	elasticCmd.Flags().StringP("es_address", "a", "http://localhost:9200", "url for Elasticsearch server")
```

Comment:
you can abreviate this to `addr` instead of `es_address`. and the description can be `address for Elasticsearch server`